torch/__init__.py

```diff
@@ -288,6 +288,12 @@ def _preload_cuda_deps(lib_folder: str, lib_name: str) -> None:
         candidate_lib_paths = glob.glob(
             os.path.join(nvidia_path, lib_folder, "lib", lib_name)
         )
+        # if path/nvidia/lib_folder/ is not found look in path/lib_folder/
+        if not candidate_lib_paths:
+            candidate_lib_paths = glob.glob(
+                os.path.join(path, lib_folder, "lib", lib_name)
+            )
```

Comment:
How this is related to pre-loading cusparselt?

Reply:
Basically looks like cusparselt is installed into 
```
/usr/local/lib/python3.9/site-packages/cusparselt/lib/
```
rahter then 
```
/usr/local/lib/python3.9/site-packages/nvidia/cusparselt/lib/
```